precode.go

```diff
@@ -68,7 +101,10 @@ func main() {
 	var sum int64   // сумма чисел результирующего канала
 
 	// 5. Читаем числа из результирующего канала
-	// ...
+	for v := range chOut {
+		count++
+		sum = sum + v
```

Comment:
Можно записать это короче
```suggestion
		sum += v
```